src/components/data-management/ProjectDetails.jsx

```diff
@@ -577,22 +622,30 @@ const ProjectDetails = ({ width, height }) => {
 
   const DownloadDataMenu = (
     <Menu>
-      <Menu.Item disabled>
-        <Tooltip title='Coming soon!' placement='left'>
+      <Menu.Item disabled key='download-raw-seurat'>
+        <Tooltip title='Feature coming soon!' placement='left'>
           {/* <Tooltip title='Samples have been merged' placement='left'> */}
           Raw Seurat object (.rds)
         </Tooltip>
+        s
       </Menu.Item>
-      <Menu.Item disabled>
-        <Tooltip title='Coming soon!' placement='left'>
-          {/* <Tooltip title='Data Processing filters have been applied' placement='left'> */}
-          Processed Seurat object (.rds)
+      <Menu.Item
+        key='download-processed-seurat'
+        disabled={!pipelineHasRun}
+      >
+        <Tooltip title='Data Processing filters have been applied' placement='left'>
```

Comment:
I think this tooltip is not very clear. I would probably write something like "Seurat object produced by applying data processing filters" (not very happy with that  phrasing either. Do you have any suggestion @vickymorrison ?

Reply:
"Download data with filters applied"?

Reply:
How about "With Data Processing filters and settings applied"?